tls/s2n_cipher_preferences.c

```diff
@@ -181,6 +181,35 @@ const struct s2n_cipher_preferences cipher_preferences_20170210 = {
     .minimum_protocol_version = S2N_TLS10
 };
 
+/* Preferences optimized for interop. DHE and 3DES are added(at the lowest preference). */
```

Comment:
I think we can add it. The approach I used in #355 has us skip ciphers in the preference list that the libcrypto won't support.